components/Home/Donate/index.jsx

```diff
@@ -180,6 +204,15 @@ export const DepositServiceDonation = () => {
             <Paragraph>{e.value}</Paragraph>
           </EpochStatus>
         ))}
+
+        <Button
+          type="primary"
+          loading={isCheckpointLoading}
+          disabled={isExpectedEndTimeInFuture}
+          onClick={onCheckpoint}
+        >
+          Checkpoint
```

Comment:
```suggestion
          Start new epoch
```

Reply:
Internally maybe it makes sense to refer to checkpoint, but it's not explicit for an external user what this means.